workflow-core/src/main/java/com/squareup/workflow/WorkflowPool.kt

```diff
@@ -89,9 +89,8 @@ class WorkflowPool {
    * Unique identifier for a particular [Workflow] to be run by a [WorkflowPool].
    * See [Type.makeWorkflowId] for details.
    *
-   * Convenience extension functions exist on `KClass<Launcher>` and [Delegating] to create IDs:
+   * A convenience extension function exists on `KClass<Launcher>` to create IDs:
    *  - `KClass<Launcher>.makeWorkflowId()`
```

Comment:
Nit: if this is just the only item in the bullet list now, doesn't need to be a list.